src/helpers.coffee

```diff
@@ -226,7 +232,7 @@ keyForKeyboardEvent = (event, dvorakQwertyWorkaroundEnabled) ->
   if process.platform in ['linux', 'win32']
     keyIdentifier = translateKeyIdentifierForWindowsAndLinuxChromiumBug(keyIdentifier)
 
-  return null if KeyboardEventModifiers.has(keyIdentifier)
+  return keyIdentifier if KeyboardEventModifiers.has(keyIdentifier)
```

Comment:
Soooo this is a weird change. `keyup` events for modifier keys will have the corresponding boolean to _false_, because, I mean, you just lifted the key. So this will pass the actual text back, and there is some corresponding code elsewhere to not duplicate the keystroke (e.g. so you dont get `ctrl-ctrl`).